src/java.desktop/share/classes/sun/swing/SwingUtilities2.java

```diff
@@ -1706,10 +1704,8 @@ private static Object makeIcon(final Class<?> baseClass,
                                   final String imageFile,
                                   final boolean enablePrivileges) {
         return (UIDefaults.LazyValue) (table) -> {
-            @SuppressWarnings("removal")
-            byte[] buffer = enablePrivileges ? AccessController.doPrivileged(
-                    (PrivilegedAction<byte[]>) ()
-                    -> getIconBytes(baseClass, rootClass, imageFile))
+            byte[] buffer = enablePrivileges ?
+                    getIconBytes(baseClass, rootClass, imageFile)
```

Comment:
```suggestion
            byte[] buffer = getIconBytes(baseClass, rootClass, imageFile);
```

Maybe you could move the method body into `makeIcon(Class<?>, Class<?>, String)`

Reply:
This enablePrivileges param is interesting and obsolete.
Either here or in an immediate follow-on we should get rid of it.
I'm inclined to do it in a small follow-on, not here.
javax.swing.LookAndFeel.makeIcon(.) is the only place that calls
 SwingUtilities2.makeIcon_Unprivileged so it won't be hard.

Reply:
I submitted https://bugs.openjdk.org/browse/JDK-8344569

Reply:
ok..you can assign to me if you want..